codecs/opus/src/test/java/org/restcomm/media/codec/opus/OpusCodecTest.java

```diff
@@ -42,6 +50,9 @@
  * 
  */
 public class OpusCodecTest implements OpusJni.Observer {
```

Comment:
Test class should not implement OpusJni.Observer

Reply:
Done